requirements.txt

```diff
@@ -1,6 +1,6 @@
 packaging==23.1
 torch==2.0.0
-openai==0.27.0
+openai==1.14.1
```

Comment:
All my comments are based on the knowledge of 1.0.1, so please let me know if things have change, e.g. dealing with Azure or the err_type.